.travis.yml

```diff
@@ -6,11 +6,9 @@ script: "bundle exec rake validate && bundle exec rake lint && bundle exec rake
 matrix:
   fast_finish: true
   include:
-  - rvm: 1.8.7
-    env: PUPPET_GEM_VERSION="~> 3.0"
   - rvm: 1.9.3
     env: PUPPET_GEM_VERSION="~> 3.0"
-  - rvm: 2.0.0
```

Comment:
This would remove support for Ruby 1.8.7 and 2.0.0 though the README states that those are supported.